rclpy/topics/minimal_subscriber/subscriber_local_function.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+import rclpy
+
+from std_msgs.msg import String
+
+
+def main(args=None):
+    if args is None:
+        args = sys.argv
+
+    rclpy.init(args)
+
+    node = rclpy.create_node('minimal_publisher')
+
+    def listener_callback(msg):
+        print('I heard: [%s]' % msg.data)
+
+    subscription_ = node.create_subscription(
+        String, 'topic', listener_callback)
+    subscription_  # noqa
```

Comment:
This line tells the linter that this variable is not an unused variable. That's the counterpart of calling the destroy_* functions explicitely.